apps/vaporgui/main.cpp

```diff
@@ -71,8 +73,49 @@ FILE *OpenLog(string path_var) {
 	return(fp);
 }
 
+
+struct opt_t {
+    OptionParser::Boolean_T    help;
+    OptionParser::IntRange_    renderRange;
+    OptionParser::Dimension2D_ resolution;
+    char*                      outputPath;
+} opt;
+OptionParser::OptDescRec_T    set_opts[] = {
+    {"help",       0,    "",           "Print this message and exit"},
+    {"render",     1,    "0:0",        "Render timesteps a:b for a given session file and exit"},
+    {"resolution", 1,    "1920x1080",  "Output resolution when using -render"},
+    {"output",     1,    "vapor.tiff", "Output image file when using -render. This will be suffixed with the timestep number. Supports tiff, jpg"},
+    {NULL}
+};
+OptionParser::Option_T    get_options[] = {
+    {"help",       Wasp::CvtToBoolean,     &opt.help, sizeof(opt.help)},
+    {"render",     Wasp::CvtToIntRange,    &opt.renderRange, sizeof(opt.renderRange)},
```

Comment:
It might improve usability to separate the render option into a flag that specifies "perform batch rendering", and a separate flag for specifying the time steps, with the default being all available time steps. That way the user doesn't have to figure out how many time steps are in a data collection.

Reply:
Done.
```
> vapor -help
Usage: vapor [options] [session.vs3] [data files...]
    OPTION        NUM_ARGS    DEFAULT
    ------        --------    -------
    -help         0           false
        Print this message and exit
    -output       1           vapor.tiff
        Output image file when using -render. This will be suffixed with the
        timestep number. Supports tiff, jpg
    -render       0           false
        Render a given session file and exit
    -resolution   1           1920x1080
        Output resolution when using -render
    -timesteps    1           0:0
        Timesteps to render when using -render. Defaults to all timesteps.
```